src/sdk/nb.d.ts

```diff
@@ -63,6 +63,7 @@ interface System extends Base {
 
 interface Account extends Base {
     _id: ID;
+    owner: ID;
```

Comment:
do you know if owner is required for account? maybe @shirady can help
in the nsfs_account_schema it doesn't. but let's keep it the same there and here.

Reply:
not required in both schemas:
https://github.com/noobaa/noobaa-core/blob/1382037a986722a59a07346bd140243f21e11ead/src/server/system_services/schemas/nsfs_account_schema.js#L7-L16

https://github.com/noobaa/noobaa-core/blob/1382037a986722a59a07346bd140243f21e11ead/src/server/system_services/schemas/account_schema.js#L9-L14

```suggestion
    owner?: ID;
```